bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx

```diff
@@ -29,34 +31,9 @@ export default class ActionsBar extends Component {
           <EmojiContainer />
         </div>
         <div className={styles.hidden}>
-          <ActionsDropdown />
-        </div>
-      </div>
-    );
-  }
-
-  renderForUser() {
-    return (
-      <div className={styles.actionsbar}>
-        <div className={styles.center}>
-          <MuteAudioContainer />
-          <JoinAudioOptionsContainer
-            handleJoinAudio={this.props.handleOpenJoinAudio}
-            handleCloseAudio={this.props.handleExitAudio}
-
-          />
-          {/*<JoinVideo />*/}
-          <EmojiContainer />
+          <ActionsDropdown {...{isUserPresenter}}/>
```

Comment:
The second <ActionsDropdown> is there to assist in the positioning of the other buttons. Without it the rest of the buttons on the actions bar do not line up symmetrically with the rest of the page.